### PR TITLE
add timeZone param for timeline, remove maxSize params

### PR DESCRIPTION
**Pull request by mikenevi**

### CODE REVIEW

File: Example/healthvault-ios-sdk/ViewControllers/Timeline/MHVTimelineSnapshotViewController.m [hunk 1]
```diff
@@ -68,9 +68,10 @@ - (void)loadTimeline
     [self.statusLabel showBusy];
     
     MHVLocalDate* startDate = [[MHVLocalDate alloc] initWithObject:@"2017-05-29" objectParameters:nil];
```
Comment (miburf) [inline Example/healthvault-ios-sdk/ViewControllers/Timeline/MHVTimelineSnapshotViewController.m]: Maybe update to be the past 60 days or something?  Running the app a couple years from now could be a lot of data

File: HealthVault/Classes/Models/Generated/Categories/MHVTaskTrackingApi.m [hunk 2]
```diff
@@ -40,6 +40,109 @@ @implementation MHVRemoteMonitoringClient (MHVTaskTrackingApi)
...
+{
+    // verify the required parameter 'occurrenceId' is set
+    if (occurrenceId == nil)
+    {
+        NSParameterAssert(occurrenceId);
+        if(completion)
+        {
+            NSDictionary * userInfo = @{NSLocalizedDescriptionKey : [NSString stringWithFormat:NSLocalizedString(@"Missing required parameter '%@'", nil),@"occurrenceId"] };
+            NSError* error = [NSError errorWithDomain:kMHVTaskTrackingApiErrorDomain code:kMHVTaskTrackingApiMissingParamErrorCode userInfo:userInfo];
+            completion(nil, error);
```
Comment (miburf) [inline HealthVault/Classes/Models/Generated/Categories/MHVTaskTrackingApi.m:60]: should return.  This'll then do the request, so would call the completion twice

**Review by miburf — APPROVED** [review]
A few things to fix

File: HealthVault/Classes/Models/Generated/Categories/MHVTaskTrackingApi.m (hunk 2 above)
Comment (mikenevi) [reply HealthVault/Classes/Models/Generated/Categories/MHVTaskTrackingApi.m:60]: Don't know how we didn't notice this before 👍

**Pull request merged**